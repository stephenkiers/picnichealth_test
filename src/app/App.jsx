import React, { Component } from 'react'
import RecordsContainer from "./Record/index/RecordsContainer";
import NewRecordButtonContainer from "./components/NewRecordButton/NewRecordButtonContainer";
import UpdateRecordModalContainer from "./Record/update_modal/UpdateRecordModalContainer";
import StatsContainer from "./AppState/Stats/StatsContainer";

class App extends Component {
    render () {
        return (
            <div>
                <header>
                    <h1>
                        <img src="https://palcehold.it/50x50" />
                        <span>
                            React Test Application
                        </span>
                    </h1>
                </header>
                <div className="container">
                    <div className="universal-actions clearfix">
                        <div className="pull-right">
                            <StatsContainer />
                        </div>
                        <div className="pull-left">
                            <NewRecordButtonContainer />
                        </div>
                    </div>
                    <RecordsContainer />
                </div>
                <UpdateRecordModalContainer />
            </div>
        )
    }
}

export default App;