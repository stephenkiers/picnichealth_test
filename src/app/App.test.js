import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import App from './App';

test('App displays correct', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(
        <App />
    );
    const tree = renderer.getRenderOutput()
    expect(tree).toMatchSnapshot();
});