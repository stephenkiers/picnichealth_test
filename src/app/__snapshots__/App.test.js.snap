// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App displays correct 1`] = `
<div>
  <header>
    <h1>
      <img
        src="https://www.granify.com/hs-fs/hub/315877/file-2096520345-png/images/granify-logo-white-1.png?t=1499980498886"
      />
      <span>
        Javascript Test Application
      </span>
    </h1>
  </header>
  <div
    className="container"
  >
    <div
      className="universal-actions clearfix"
    >
      <div
        className="pull-right"
      >
        <Connect(StatsContainer) />
      </div>
      <div
        className="pull-left"
      >
        <Connect(NewRecordButtonContainer) />
      </div>
    </div>
    <Connect(RecordsContainer) />
  </div>
  <Connect(UpdateRecordModalContainer) />
</div>
`;
